docker-hub/scim.md

```diff
@@ -1,42 +1,44 @@
+| givenName                            | User’s first name |
+| familyName |User’s surname                                              |
+
+## Configure
 
-### Okta IdP
+SAML-based SSO must be properly set up and be functional for your organization before you start configuring automated provisioning. In addition, you must verify your company domain and have the appropriate connectors in your IdP. Your user email domain must be the same company domain you use for Single Sign-on (SSO). Enforcing SSO is not required to enable SCIM. However, you must [configure SAML SSO](../single-sign-on/index.md){: target="_blank" rel="noopener" class="_"} before you enable SCIM.
 
-Before making SCIM configuration changes in your IdP, navigate to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} and select **Organizations** > **Settings** > **Security**. SCIM is locked until you complete the SSO configuration and verify your company domain. Enable SCIM and access your **Base URL** and **API Token** ( the same as Bearer Token in Okta).
+Before you make SCIM configuration changes in your IdP, navigate to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} and select **Organizations** > **Settings** > **Security**. SCIM is locked until you complete the SSO configuration and verify your company domain. Enable SCIM and access your Base URL and API Token. You can also generate a new API token.
 
 ![scim-provisioning](images/scim-provisioning.png){:width="700px"}
 
-1. In Okta, navigate to **Applications** > **Create App Integration**, **SAML 2.0**, and click **Next**.
-2. In the **General** tab, select **Edit App Settings** to enable SCIM provisioning and click **Save**.
-3. In the Provisioning tab, edit the SCIM Connection and complete the following:
+### Okta
+
+1. In Okta, navigate to **Applications** > **Create App Integration** > **SAML 2.0**, and click **Next**.
+2. In the **General** tab, on your new application page, select **Edit App Settings** to enable SCIM provisioning and click **Save**.
+3. In the **Provisioning** tab, edit the SCIM Connection and complete the following:
 
     * **SCIM connector base URL**: SCIM Base URL from Docker Hub
     * **Unique identifier field for users**: enter **email**
     * **Supported Provisioning actions**: select **Push New Users**, **Push Profile Updates**
     * **Authorization/Bearer**: SCIM API Token from Docker Hub
 
 4. Click **Test Connection Configuration** to complete the configuration and **Save**.
-5. Navigate to **Provisioning** > **To App** > **Edit** and enable **Create Users**, **Update User Attributes** and **Deactivates Users**, and click **Save**.
+5. Once configured, you must enable synchronization. Navigate to **Provisioning** > **To App** > **Edit**, and enable **Create Users**, **Update User Attributes** and **Deactivates Users**, and click **Save**.
 
     ![scim-app-provisioning](images/scim-app-provisioning.png){:width="700px"}
```

Comment:
suggestion: This screenshot is more relevan to the step 3. Move this image to the step 3, then use [this image](https://www.notion.so/dockerinc/TC004-Configure-SCIM-on-Okta-4bccb09190614dd89cb65fd74bb55bb7#abb05bd592af4ed2b4d55eefec6837a6) for the step 5

Reply:
Updated step 3 and step 5 in new commit